[Bug]: AWS API requests are getting failed due to change in parameters

### Description
AWS API requests are getting failed due to change in parameters

`E               Error is fatal error: An error occurred (PermanentRedirect) when calling the ListObjectsV2 operation: The bucket you are attempting to access must be addressed using the specified endpoint. Please send all future requests to this endpoint: <bucket_name>.s3.amazonaws.com
E               You can fix this issue by explicitly providing the correct region location using the --region argument, the AWS_DEFAULT_REGION environment variable, or the region variable in the AWS CLI configuration file.  You can get the bucket's location by running "aws s3api get-bucket-location --bucket BUCKET".
E               command terminated with exit code 1`

RP link: https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#ocs/launches/852/32463/1592833/1592988/log

--- a/ocs_ci/ocs/constants.py
+++ b/ocs_ci/ocs/constants.py
@@ -2571,7 +2571,7 @@ NETWORK_FENCE_CLASS_CRD = os.path.join(
 NETWORK_FENCE_CRD = os.path.join(TEMPLATE_CSI_ADDONS_DIR, "network-fence-rbd.yaml")
 
 # MCG namespace constants
-MCG_NS_AWS_ENDPOINT = "https://s3.amazonaws.com"
+MCG_NS_AWS_ENDPOINT = f"https://s3.{DEFAULT_AWS_REGION}.amazonaws.com"
 MCG_NS_AZURE_ENDPOINT = "https://blob.core.windows.net"
 MCG_NS_RESOURCE = "ns_resource"
 MCG_NSS = "ns-store"

